doc/setup.md

```diff
@@ -115,28 +145,22 @@ When developing with SkyPortal on mac, you may  also need to configure your shel
 ```
 nano ~/.zshrc
 ```
-#### Set enviroment variables to their installation paths
-After installing the libraries with Homebrew, you'll need to set environment variables to their installation paths. Replace the placeholder text `<path-to-library>/<version>` with the actual path that Homebrew provides upon sucessful installation.
+#### Set environment variables to their installation paths
+After installing the libraries with Homebrew, you'll need to set environment variables to their installation paths. Replace the placeholder text `<path-to-library>/<version>` with the actual path that Homebrew provides upon successful installation.
 
 ```
 export HDF5_DIR="<path-to-hdf5>"
 export BLOSC_DIR="<path-to-c-blosc>"
 export LZO_DIR="<path-to-lzo>"
 export BZIP_DIR="<path-to-bzip2>"
 ```
+If you have forgotten your library install paths, you can run `brew info <package_name>` to display the install path.
```

Comment:
I don't think this shows the version number that's in the path, right? If not, please explain how to get it (with ls)

Reply:
I would replace "have forgotten" by "did not take note of"